hole-punch-interop/versions.ts

```diff
@@ -10,9 +10,8 @@ export type Version = {
 
 export const versions: Array<Version> = [
     {
-        id: "rust-master",
-        transports: ["tcp", "quic"],
-        containerImageID: readImageId("./impl/rust/master/image.json"),
```

Comment:
I am assuming that this is a left-over @thomaseizinger?

Reply:
What do you mean with left-over?

Reply:
I am surprised that the field is removed (not altered) and thus assume that is not actually needed in the first place. Is that not the case @thomaseizinger?

Reply:
The field was needed because the naming of the version ("master") did not conform with the default naming pattern of semver and thus the default resolution function of the JSON file would not have worked.